skafka/src/main/scala/com/evolutiongaming/skafka/producer/ProducerLogging.scala

```diff
@@ -51,6 +54,21 @@ object ProducerLogging {
       def partitions(topic: Topic) = producer.partitions(topic)
 
       def flush = producer.flush
+
+      private def logError[K, V](record: ProducerRecord[K, V], err: Throwable): F[Unit] =
+        err match {
+          case _: RecordTooLargeException =>
+            val trimmed = record.value.map(_.toString().take(charsToTrim))
+
+            log.error(
+              s"Failed to send too large record: topic = ${record.topic}, " +
```

Comment:
across library we use semantic like `Failed to send too large record, topic: …`, could you please make it unified

Reply:
Done